package-lock.json

```diff
@@ -7235,8 +7235,7 @@
             },
             "ansi-regex": {
               "version": "2.1.1",
-              "bundled": true,
-              "optional": true
+              "bundled": true
```

Comment:
Yes probably because i run an "npm install" after pulling from master. I was getting an error with the date picker component Lambert created however with the npm install, the portal was able to load.